.code-samples.meilisearch.yaml

```diff
@@ -68,6 +68,25 @@ delete_documents_1: |-
   })
 search_post_1: |-
   client.Index("movies").Search("american ninja", &meilisearch.SearchRequest{})
+multi_search_1: |-
+  client.MultiSearch(&MultiSearchRequest{
```

Comment:
I'm not that very good with go but is it possible that there is a missing `queries` field here?